probe-rs/src/probe/stlink/mod.rs

```diff
@@ -1646,6 +1646,8 @@ impl ArmProbe for StLinkMemoryInterface<'_> {
     fn ap(&mut self) -> MemoryAp {
         self.current_ap
     }
+
+    fn update_core_status(&mut self, _: crate::CoreStatus) {}
```

Comment:
Could give this a default impl on the trait to avoid writing this signature over and over :)

Reply:
Oops, I somehow missed that it was right next to `get_arm_communication_interface` which is the only thing using that ATM. Fixed!